wiki/Beatmap_Discussion/en.md

```diff
@@ -126,14 +126,22 @@ This is where you submit your mod. You can choose between Praise, Suggestion, or
 ### Functions
 
 - `Collapse all`/`Expand all` and the up arrow: These will show/hide posts.
-- Thumbs Up/Down: Like/Dislike the mod. Everyone but the mod submitter can use this.
+- Thumbs Up/Down: Like/Dislike the mod. Everyone but the mod submitter can use this. **Giving a thumbs up will now give the mod submitter [Kudosu!](/wiki/Kudosu!) on suggestions and problems.**
 - Timeline: Shows the timestamp the issue addresses. Only available in the Timeline section.
 - CREATOR icon: Displays responses/submissions from the user who made the beatmapset.
 - permalink, edit, delete: Permalink grabs a direct link to the post. Edit allows the submitter to edit their submission. Delete allows the submitter to remove their submission.
 - Respond: Reply to the submission. Pressing Enter or clicking Reply afterwards will submit your response.
 - Mark as Resolved: The creator may select this when the issue has been addressed and either accepted or denied.
 - Reopen a Discussion: If the submission needs further discussion, the creator or mod submitter may reopen the submission by responding and unchecking Resolved.
 
+## Earning Kudosu!
+
+*Note: You are given ten votes every hour.*
+
+![](img/thumbs.jpg "Thumbs up for Kudosu!")
+
+Modders can now earn Kudosu! by having their mod be given the thumbs up. You can (and probably should) go through your old mods and give them a thumbs up to support the modders before this feature was implemented.
```

Comment:
by having a thumbs up on their mod

through your old mods -> through mods on your old beatmaps?

Reply:
applied `by having a thumbs up on their mod`
...
`through your old mods` should be good enough, its talking about the mods before Kudosu! was implemented in beatmap discussion (not anything regards to old beatmaps)

Reply:
`through you old mods and give them a thumbs up` it means that you give thumb up to **your** own mods

Reply:
not other modders' mods

Reply:
alright, using `go through mods on your beatmaps` (your second suggestion, but removed `old` before `beatmaps`)